src/main/kotlin/gropius/model/template/MutableTemplatedNode.kt

```diff
@@ -0,0 +1,12 @@
+package gropius.model.template
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.annotations.GraphQLIgnore
+
+@GraphQLDescription("Interface for all types which support templates that can be written by the user.")
```

Comment:
technically, not the template, but the templated fields can be written by the user

Reply:
627fa894d0487f997628d7d8f967a37420cce136?